canonical-websites/vanillaframework.io
# Add ./run script

QA
--

``` bash
./run
```

Browse to <http://127.0.0.1:8018>, check it works.

Try some other things:

``` bash
./run clean
./run watch
./run serve -p <port>
./run serve -w
./run server --detach
./run build
./run test
```
